test/e2e/tests/003-surveyInfoEdit.js

```diff
@@ -0,0 +1,66 @@
+import {
+  button,
+  clearTextBox,
+  click,
+  expectExists,
+  writeIntoTextBox,
+  hoverTextBox,
+  toRightOf,
+  waitFor1sec,
+} from '../utils/api'
+import { clickHomeBtnEditSurveyInfo, waitForLoader, clickSidebarBtnHome } from '../utils/ui'
+
+const verifyHomeDashboard = async ({ label }) => {
+  await expectExists({ selector: '.home-dashboard' })
+  await expectExists({ text: label })
+}
```

Comment:
isn't the same in survey create test? if yes, move it to a common ui/home module and call the method expectHomeDashboard